doc/source/features/strict-mode.rst

```diff
@@ -0,0 +1,296 @@
+.. _strict-mode:
+
+===========
+Strict Mode
+===========
+
+This mode is designed to avoid problems with improper configuration of |PXC|.
+It performs a number of validations against essential features of |PXC|.
+Some validations are performed only at startup,
+while others remain enabled during runtime.
+
+Depending on the actual mode you select,
```

Comment:
Just easier to edit compared to when each paragraph is on the same line or simply hard wrapping at 80 chars. Breaking lines at logical "blocks" of text, like when a new sentence is started or after a comma that defines two parts of a sentence. When I need to rephrase a part of the sentence, I can simply replace the whole line, because it is a separate logical part of the sentence. Sort of like treating text as code.